README.md

```diff
@@ -15,17 +15,16 @@ Modify `kafka-faas-connector.yml` to set the correct URLs of the OpenFaaS gatewa
 
 ### Local execution
 
-Launch Kafka and ZooKeeper or establish a connection to already running instances (e.g. via port forwarding).
+Launch Kafka and ZooKeeper locally. Either start OpenFaaS also locally or establish a connection to an already running instance (e.g. via port forwarding).
 
 docker-compose:
 ```bash
 docker-compose up kafka zookeeper
 ```
 
-Port forwarding:
+Port forwarding to OpenFaaS:
 ```bash
-kubectl port-forward svc/kafka -n openfaas 9092
-kubectl port-forward svc/zookeeper -n openfaas 2181
+kubectl port-forward svc/gateway -n openfaas 31112:8080

```

Comment:
why delete the commands to port forward existing kafka?
Maybe keep them for convenience.

Reply:
Port-forwarding to Kafka does basically not work. See [this](https://medium.com/@valercara/connecting-to-a-kafka-cluster-running-in-kubernetes-7601ae3a87d6) blog post for more information.

I tried many different variants, finally i came to the conclusion, that the best way is to choose one of the following options:
* Local: Execute Kafka, ZooKeeper and KafkaFaasConnector locally via `docker-compose` and run the Kafka scripts (e.g. to produce oder consume messages) inside the Kafka container (`docker exec ...`)
* Kubernetes: Everything runs inside the cluster. The Kafka scripts can be executed inside the Kafka pod (`kubectl exec`) or by using *kafkacat* as a separate container.

For more information see my notes [here](https://mico-docs.readthedocs.io/en/latest/setup/kubernetes/kafka.html).